Backend/Backend/Repository/UsersRepository.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Backend.Repository.Interfaces;
+using DataAccess.Data;
+using DataAccess.Data.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace Backend.Repository
+{
+    public class UsersRepository : IUsersRepository
+    {
+        private readonly ApplicationDbContext _dbContext;
+
+        public UsersRepository(ApplicationDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        public async Task<int> Create(string firstName, string lastName, int age)
+        {
+            var user = await _dbContext.Users.AddAsync(new User { FirstName = firstName, LastName = lastName, Age = age });
+            await _dbContext.SaveChangesAsync();
+            return user.Entity.Id;
+        }
+
+        public async Task<bool> Delete(int id)
+        {
+            try
+            {
+                _dbContext.Users.Remove(await _dbContext.Users.FirstOrDefaultAsync(x => x.Id == id) ?? throw new NullReferenceException("User wasn't found"));
+                await _dbContext.SaveChangesAsync();
```

Comment:
Only one error handled. What about the rest of the errors associated with the methods? Maybe it's better to handle errors at another level of abstraction? Or you can process them right away, but handle all.

Reply:
I don't understand clearly what do you mean. Is it bad to handle only one error here? Which errors i have to handle else?